tech-specs/src/explore/design/pos.md

```diff
@@ -0,0 +1,283 @@
+# Proof of Stake (PoS) system
+
+## Epoch
+
+An epoch is a range of blocks or time that is defined by the base ledger and made available to the PoS system. This document assumes that epochs are identified by consecutive integers. All the data relevant to PoS are [associated with epochs](#epoched-data).
```

Comment:
They should be natural numbers, as epochs will never be negative

Reply:
updated in b72f7c92f03928e95a845342ae8a88cb8dad8458